<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysOverviewObjectShapeManagerWidget</class>
 <widget class="QWidget" name="SysOverviewObjectShapeManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="shapeLbl">
     <property name="text">
      <string>Shape</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="shapeComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="colorLbl">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="colorLE">
       <property name="inputMask">
        <string>\#HHHHHH</string>
       </property>
       <property name="text">
        <string>#000000</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorPickerBtn">
       <property name="text">
        <string>Color Picker</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="transparencyLbl">
       <property name="text">
        <string>Transparency</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="transparencySpinBox">
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
